loader.php

```diff
@@ -18,14 +18,14 @@ function is_development() {
  * Attempt to load a file at the specified path and parse its contents as JSON.
  *
  * @param string $path The path to the JSON file to load.
- * @return array|null;
+ * @return string|null;
```

Comment:
It does return an array or null. See `json_decode` at the end of the function.

Reply:
Ah, I somehow read ENcode... Will revert the type.